docs/index.rst

```diff
@@ -6,8 +6,8 @@ A sphinx extension for creating panels in a grid layout.
 
 This directive creates panels of content in a grid layout,
 utilising both the bootstrap 4
-`grid system <https://www.w3schools.com/bootstrap/bootstrap_grid_system.asp>`_,
-and `cards layout <https://www.w3schools.com/bootstrap4/bootstrap_cards.asp>`_.
+`grid system <https://getbootstrap.com/docs/4.0/layout/grid/>`_,
```

Comment:
Is there any reason to think we'd run into issues if other bootstrap CSS/JS gets loaded on top of the CSS rules in here? (other than some unnecessary data transfer)

Reply:
This is why you can set `add_boostrap_css = False` in `conf.py`, then no CSS will be loaded by this extension. Then there should be no issue?

Reply:
ah right - I agree. Can we add a short prefix to that configuration value? E.g. `panels_add_bootstrap_css`, as Sphinx suggests?

Reply:
fixed in 0a312cd

Reply:
another thought here is that if we had a "meta" extension that loaded bootstrap, that extension could be configured selectively in a single variable, rather than in a bunch of one-off configurations. e.g.:

```
sphinx_bootstrap_load_css = ["panel", "grid"]
```

etc. Or just `"all"` for all CSS classes